azurerm/internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -296,3 +344,17 @@ func flattenKustoClusterSku(sku *kusto.AzureSku) []interface{} {
 
 	return []interface{}{s}
 }
+
+func flatteKustoClusterVNET(vnet *kusto.VirtualNetworkConfiguration) []interface{} {
+	if vnet == nil {
+		return []interface{}{}
+	}
+
+	output := map[string]interface{}{
+		"subnet_id":                    string(*vnet.SubnetID),
```

Comment:
We'll want to check all these aren't nil before pointing to them. And just to call out what the linter said about not needing to cast these to a string.

Reply:
Fixed